### ISSUE TITLE
Problem with the overall GUI

### ISSUE THREAD

**Issue by Jimmy-Blue** [lapce/lapce#1083]
## Lapce Version

v0.2.0

## System information

Windows 10 21H2 build 19044.1889

## Describe the bug
The window itself is off-centered, which make buttons hard to click, navigate, etc. More details can be seen from the video.

## Additional information

I think it is a GUI related problem? I have no idea then.

- screenshots

https://user-images.githubusercontent.com/60958064/188929503-f996f635-dfa5-4308-b3d1-dc929d044d76.mp4

![image](https://user-images.githubusercontent.com/60958064/188927833-61288a51-d795-47fa-b6b2-d41f9899a49f.png)

- logs
No idea.

- theories about what might be going wrong
Probably a bug with GUI? It looks like Gtk, from my experience.

- workarounds that you used
Just close and reopen without any fix.

**Comment by panekj** [comment]
Does it change anything if you try disabling/enabling custom titlebar in settings?

**Comment by Jimmy-Blue** [comment]
Apparently, it does fix the problem, but seems like a bug or something?

**Comment by panekj** [comment]
> but seems like a bug or something?

yes, and I'm trying to figure out why it happens..

does it happen still after closing/reopening lapce?

**Comment by Jimmy-Blue** [comment]
> > but seems like a bug or something?
> 
> yes, and I'm trying to figure out why it happens..
> 
> does it happen still after closing/reopening lapce?

With custom tilte bar enabled, it does, but with that disabled, not.

**Comment by superlou** [comment]
I wonder if this is related to #1014. I tried disabling custom titlebar and the symptoms from #1014 go away. Maybe the whole app in that case is just slid upwards?

**Comment by panekj** [comment]
> I wonder if this is related to #1014. I tried disabling custom titlebar and the symptoms from #1014 go away. Maybe the whole app in that case is just slid upwards?

I cannot reproduce missing titlebar but I can reproduce that offset all the time when using Intel HD Graphics. It also doesn't happen when launching Lapce with NVidia GPU.

**Comment by Jimmy-Blue** [comment]
> > I wonder if this is related to #1014. I tried disabling custom titlebar and the symptoms from #1014 go away. Maybe the whole app in that case is just slid upwards?
> 
> I cannot reproduce missing titlebar but I can reproduce that offset all the time when using Intel HD Graphics. It also doesn't happen when launching Lapce with NVidia GPU.

Mine is running on an Intel HD Graphics as well, as I don't have a dedicated Nvidia GPU.

**Comment by superlou** [comment]
I also have Intel graphics (UHD Graphics 360).

**Comment by Curstantine** [comment]
Yep, repoducible on Intel HD 4000 with an unrelated flickering issue.

https://user-images.githubusercontent.com/69104500/218236528-8be7907c-06dc-49ea-8ce8-9b74c9ff740e.mp4

**Comment by Borwe** [comment]
> Yep, repoducible on Intel HD 4000 with an unrelated flickering issue.
> 
>  2023-02-11.08-52-02.mp4 
> Edit: The flickering issue persists even without the custom titlebar.

Yup, same issue as you, flickering when moving around.

**Comment by hex4badass** [comment]
### Suggestion: Consider disabling the "Custom Titlebar" setting by default in Windows.

As a new user trying out the app, I encountered the issue with the "Custom Titlebar" setting. This led me to reinstall the app, but the problem persisted. After reinstalling and disabling the "Custom Titlebar" setting, the app worked properly. I believe that setting it to disabled by default could improve the user experience for new users and prevent them from uninstalling the app unnecessarily.

Thank you in advance for considering this suggestion.


***I sincerely apologize for any errors or oversights I may have made in my initial attempt at posting on GitHub, as this is my first-ever attempt to do so.***

**Comment by ethereumdegen** [comment]
I have a similar issue.  The new experimental tital bar, when enabled, prevents me from resizing the window (Linux Mint) and also when not maximized, the window is stuck way up in the left corner and 90% hidden offscreen. SO i cannot use it or move it all i can do is maximize.  Using system native title bar fixes.

**Comment by panekj** [comment]
> I have a similar issue. The new experimental tital bar, when enabled, prevents me from resizing the window (Linux Mint) and also when not maximized, the window is stuck way up in the left corner and 90% hidden offscreen. SO i cannot use it or move it all i can do is maximize. Using system native title bar fixes.

This is not related.
